src/middleware/verifyJwt.ts

```diff
@@ -0,0 +1,16 @@
+import { expressjwt } from 'express-jwt';
+import { jwtOptions } from '../auth/jwtOptions';
+import type {
+  NextFunction,
+  Request,
+  Response,
+} from 'express';
+
+const expressJwtMiddleware = expressjwt(jwtOptions);
+
+export const verifyJwt = (req: Request, res: Response, next: NextFunction): void => {
+  expressJwtMiddleware(req, res, next)
+    .catch((error: unknown) => {
+      throw error;
+    });
+};
```

Comment:
This function looks very redundant. Why `catch` an error only to immediately `throw` it? In fact, why wrap the `expressJwtMiddleware` at all? Perhaps this is an artifact of the development process that led here.

Can we simplify this to:
```suggestion
export const verifyJwt = expressjwt(jwtOptions);
```